docs/3.0.0-beta.x/concepts/controllers.md

```diff
@@ -8,7 +8,7 @@ Controllers are JavaScript files which contain a set of methods called **actions
 module.exports = {
   // GET /hello
   index: async ctx => {
-    ctx.send('Hello World!');
+    return 'Hello World!';
```

Comment:
the koa default way to do this is `ctx.body = 'Hello World!';`  I would like that we use standard koa features as much as possible :)